src/translations.cpp

```diff
@@ -79,8 +79,15 @@ void set_language()
     const std::string system_lang = SystemLocale::Language().value_or( "en" );
     std::string lang_opt = get_option<std::string>( "USE_LANG" ).empty() ? system_lang :
                            get_option<std::string>( "USE_LANG" );
-    DebugLog( D_INFO, D_MAIN ) << "Setting language to: '" << lang_opt << '\'';
-    TranslationManager::GetInstance().SetLanguage( lang_opt );
+    change_language( lang_opt );
+#endif
+}
+
+void change_language( std::string &lang )
```

Comment:
You only changed declaration in header but not the implementation here.

Reply:
oh, sorry about that x_x